tests/CMakeLists.txt

```diff
@@ -28,7 +29,7 @@ if (ENABLE_GTEST)
         serac_boundary_cond.cpp
         serac_mesh.cpp
         mfem_ex9p_blockilu.cpp
-	    serac_newmark_test.cpp)
+	serac_newmark_test.cpp)
```

Comment:
spacing is wrong here and on line 73

Reply:
Approved on the contingency you fix this.

Reply:
Weird... Does make style fix CmakeLists.txt files?

Reply:
Not unless we install cmakeformat (I tried through spack, it didn’t build)